core/src/rustic_ai/core/utils/cel_expr.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Callable, Dict, Optional
+
+import celpy
+
+
+class CelExpressionEvaluator:
+    """
+    Unified CEL expression evaluator for Rustic AI.
+    Wraps cel-python to provide an easy API for evaluating expressions with context.
+    """
+
+    def __init__(self):
+        self.env = celpy.Environment()
```

Comment:
lets rename this to avoid confusion with os.env
```suggestion
        self.celpy_env = celpy.Environment()
```